streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/entity/Application.java

```diff
@@ -699,17 +706,24 @@ public boolean equals(Object o) {
 
     @Override
     public int hashCode() {
-      return Objects.hash(groupId, artifactId, version);
+      return Objects.hash(groupId, artifactId, version, classifier);
     }
 
     @Override
     public String toString() {
-      return groupId + ":" + artifactId + ":" + version;
+      return groupId + ":" + artifactId + ":" + version + ":" + classifier;
     }
 
     @JsonIgnore
     public String getPath() {
-      return getGroupId() + "_" + getArtifactId() + "-" + getVersion() + ".jar";
+      return getGroupId()
```

Comment:
if classifier is null, Will there be an exception?

Reply:
Add null value judgment